src/Differ.php

```diff
@@ -25,7 +25,7 @@
     private $outputBuilder;
 
     /**
-     * @param DiffOutputBuilderInterface $outputBuilder
+     * @param string|null|DiffOutputBuilderInterface $outputBuilder
```

Comment:
You shouldn't pass `string|null` that is just for BC/legacy support because a release was done early, as such I left the hint out as to trigger SCA warnings for people so they start updating.

Reply:
LoL, I just opened this because of https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.2/src/Differ/SebastianBergmannShortDiffer.php#L29 :P

Reply:
^_^   that class will be removed in PHP-CS-Fixer 3.0 so we should be good ;)